knexfile.js

```diff
@@ -0,0 +1,30 @@
+// looking for where is prosess.env to go
+require("dotenv").config();
+
+module.exports = {
+  development: {
+    client: "postgresql",
+    connection: {
+      connectionString: process.env.DATABASE_URL,
+    },
+    migrations: {
+      directory: __dirname + "/migrations",
+    },
+    seeds: {
+      directory: __dirname + "/seeds",
+    },
+  },
+  production: {
+    client: "postgresql",
+    migrations: {
+      directory: __dirname + "/migrations",
+    },
+    seeds: {
+      directory: __dirname + "/seeds",
+    },
+    connection: {
+      connectionString: process.env.DATABASE_URL,
+      ssl: { rejectUnauthorized: false },
+    },
+  },
+};
```

Comment:
A lot of this is the same thing repeated twice. There's a concept in programming called DRY - Don't Repeat Yourself - usually when you see the same thing copied twice, it's better to combine them. This is good because if you update one of them you need to remember to update the other one, and if you forget you might get confused!

```suggestion
const sharedConfig = {
  client: "postgresql",
  connection: {
    connectionString: process.env.DATABASE_URL,
  },
  migrations: {
    directory: __dirname + "/migrations",
  },
  seeds: {
    directory: __dirname + "/seeds",
  },
};

module.exports = {
  development: sharedConfig,
  production: {
    ...sharedConfig,
    connection: {
      ...sharedConfig.connection,
      ssl: { rejectUnauthorized: false },
    },
  },
};
```

Now it's really clear - development and production config are the same, except for one thing - the SSL flag. I don't need to read a lot of code to understand that anymore! So it's clearer what you're trying to do.